src/content/tools/pub/pubspec.md

```diff
@@ -534,7 +534,7 @@ under the `environment:` field:
 
 ```yaml
 environment:
-  sdk: '>=1.19.0 <3.0.0'
+  sdk: '^3.2.0'
   flutter: ^0.1.2
```

Comment:
While you're here, can you update this to just be a lower bound, as Nate [said](https://github.com/dart-lang/site-www/pull/5864#issuecomment-2136172953) "We should not use ^ for any environment: flutter: keys." 

```suggestion
  flutter: '>= 3.22.0'
```

Could also note that in the preceding text (only the lower bound should be specified).